src/test/resources/outputs/redshift/basics/aggregation.sql

```diff
@@ -1,5 +1,7 @@
 --#[rel-aggregation-count-00]
-SELECT COUNT(*) AS "_1" FROM "default"."T" AS "T";
+SELECT count(1) AS "_1" FROM "default"."T" AS "T";
```

Comment:
self-review: most of changes in the dialect tests are the same I did for the PartiQL -> PartiQL translation (https://github.com/partiql/partiql-scribe/pull/74)